hstrat/_auxiliary_lib/_alifestd_calc_triplet_distance_asexual.py

```diff
@@ -0,0 +1,35 @@
+import alifedata_phyloinformatics_convert as apc
+import pandas as pd
+import tqdist
+
+
+# adapted from https://github.com/mmore500/hstrat/blob/d23917cf03ba59061ff2f9b951efe79e995eb4d8/tests/test_hstrat/test_phylogenetic_inference/test_tree/_impl/_tree_quartet_distance.py
+def alifestd_calc_triplet_distance_asexual(
+    ref: pd.DataFrame, cmp: pd.DataFrame
+) -> float:
+    """Calculate the triplet distance between two trees."""
+    tree_a = apc.RosettaTree(ref).as_dendropy
```

Comment:
Now that we have an alifestd as newick and alifestd collapse unifurcations, I think we can do this without converting to dendropy.